x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/datafeed/extractor/DataExtractor.java

```diff
@@ -4,7 +4,7 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-package org.elasticsearch.xpack.core.ml.datafeed.extractor;
+package org.elasticsearch.xpack.ml.datafeed.extractor;
```

Comment:
Most notably: this interface is moved from `core` to `ml` (and it's now right next to the `DataExtractorFactory` interface, which makes sense to me)